client/config/config.go

```diff
@@ -0,0 +1,52 @@
+package config
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/burntsushi/toml"
+	"golang.org/x/oauth2"
+)
+
+// Config describes the configuration for the draupnir client
+type Config struct {
```

Comment:
As with the server config, perhaps `Load` and `Store` could be methods on this struct.